presto-main/src/test/java/com/facebook/presto/sql/planner/optimizations/TestUnion.java

```diff
@@ -63,6 +64,7 @@ public void testPartialRollupAggregationsWithUnion()
                 LogicalPlanner.Stage.OPTIMIZED_AND_VALIDATED,
                 false);
         assertAtMostOneAggregationBetweenRemoteExchanges(plan);
+        assertPlanIsFullyDistributed(plan);
```

Comment:
Can't see this method in trunk, assuming it's in an unlanded commit and works as advertised.